docs/ref/migration-operations.txt

```diff
@@ -365,6 +365,84 @@ It accepts two list of operations, and when asked to apply state will use the
 state list, and when asked to apply changes to the database will use the database
 list. Do not use this operation unless you're very sure you know what you're doing.
 
+Creating migrations that add unique fields
+===========================================
+
+Applying a migration that adds unique fields to existing rows will raise an error
+because the unique field will be generated with the same value added to all rows,
+breaking the ``unique`` constraint.
+
+Therefore, the following steps should be taken (in this example, we'll
+create a uuid field, so modify with appropriate ``default`` arg for your
```

Comment:
"... we'll add a non-nullable UUID field with a default value. Modify the respective field according to your needs)."